test/e2e/host_path.go

```diff
@@ -57,7 +57,7 @@ var _ = Describe("hostDir", func() {
 		}
 	})
 
-	It("MOD volume on tmpfs should have the correct mode", func() {
+	It("should give a MOD volume on tmpfs the correct mode", func() {
```

Comment:
'should give a volume the correct mode', although now I question whether the mode is really a function of the underlying host path.